File: JAVA/2000. Reverse Prefix of Word.java
class Solution {
    public String reversePrefix(String w, char ch) {
        int a=-1;
        String k="";
        for(int i=0;i<w.length();i++){
            if(w.charAt(i)==ch){
                a=i;
                break;
            }
        }
        for(int i=0;i<=a;i++){
            k=w.charAt(i)+k;
        }
        for(int i=a+1;i<w.length();i++)
        k+=w.charAt(i);
        return k;
    }
}

File: C++/2000. Reverse Prefix of Word.cpp
class Solution {
public:
    string reversePrefix(string w, char c) {
        for(int i=0;i<w.size();i++){
            if(w[i]==c){
            reverse(w.begin(),w.begin()+i+1);
            return w;
            }
        }
        return w;
    }
};